demos/components/src/components/Notification/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { StyledNotification, StyledButton } from './Styled';
+import { Caution, Check, Information } from '../icons';
+import { Button } from '../Button';
+
+export type Severity = 'success' | 'warning' | 'error' | 'info';
+export type Size = 'sm' | 'md' | 'lg' | 'fill';
```

Comment:
How about using enums for string types in this PR? e.g., `Severity`, `Direction`

```
# nit: Do not need to full this rule in the demo apps, but Chime SDK prefers full words over abbreviations.
enum Size {
  Small,
  Medium,
   ...
}
```

For example, using an enum value as a key is less error-prone.
```
const sizes = {
  [Size.Small]: '30rem',
  [Size.Medium]: '50rem',
  ...
};
```

Reply:
I chose types here mainly because it felt like the better developer experience, since you still get the type checking/autocomplete without the little extra boilerplate that comes with enums

Happy to switch to enums though if you think they have other advantages over types

Reply:
Right, enum props will require both JS and TS to import all enums. String types looks good to me.